utils/constants/networking.go

```diff
@@ -38,13 +38,14 @@ const (
 	DefaultOutboundConnectionTimeout       = 30 * time.Second
 
 	// Timeouts
-	DefaultNetworkInitialTimeout        = 5 * time.Second
-	DefaultNetworkMinimumTimeout        = 2 * time.Second
-	DefaultNetworkMaximumTimeout        = 10 * time.Second
-	DefaultNetworkMaximumInboundTimeout = 10 * time.Second
-	DefaultNetworkTimeoutHalflife       = 5 * time.Minute
-	DefaultNetworkTimeoutCoefficient    = 2
-	DefaultNetworkReadHandshakeTimeout  = 15 * time.Second
+	DefaultNetworkInitialTimeout                   = 5 * time.Second
+	DefaultNetworkMinimumTimeout                   = 2 * time.Second
+	DefaultNetworkMaximumTimeout                   = 10 * time.Second
+	DefaultNetworkMaximumInboundTimeout            = 10 * time.Second
+	DefaultNetworkTimeoutHalflife                  = 5 * time.Minute
+	DefaultNetworkTimeoutCoefficient               = 2
+	DefaultNetworkReadHandshakeTimeout             = 15 * time.Second
+	DefaultNoIngressValidatorConnectionGracePeriod = time.Minute * 10
```

Comment:
Just for consistency
```suggestion
	DefaultNoIngressValidatorConnectionGracePeriod = 10 * time.Minute
```

Reply:
[Fixed](https://github.com/ava-labs/avalanchego/pull/3719/commits/23850fd3a9b753dd73bcf7e473bdf8fb5bfe5d9e#diff-0f86939a30a343f4c6e9de6b3efcbdbee14db63808ee96a1a1d6669cb7dab71aR48).